apps/condo/domains/organization/integrations/sbbol/SbbolFintechApi.js

```
@@ -230,6 +230,113 @@ class SbbolFintechApi extends SbbolRequestApi {
+     *          "debitBlockedTaxAuthorityCode":null,
+     *          "debitBlockedBeginDate":null,
+     *          "debitBlockedEndDate":null,
+     *          "creditBlocked":false,
+     *          "creditBlockedCause":null,
+     *          "creditBlockedInitiator":null,
+     *          "creditBlockedTaxAuthorityCode":null,
+     *          "creditBlockedBeginDate":null,
+     *          "creditBlockedEndDate":null,
+     *          "blockedSums":[
+     *          ],
+     *          "blockedQueuesInfo":[
+     *          ],
+     *          "blockedSumQueuesInfo":[
+     *          ]
+     *       },
+     *  ]
+     */
+
+    /**
+     * Get state of payment request, posted with `postPaymentRequest`
+     *
+     * @return {Promise<{error: any}|{data: ClientAccounts}>}
+     */
+    async getClientAccounts () {
+        const { data, statusCode } = await this.request({
+            method: 'GET',
+            path: this.getClientAccountsRequestPath,
+        })
+        const jsonData = JSON.parse(data)
+        if (statusCode === 200) {
+            return { data: jsonData }
+        } else {
+            return { error: jsonData }
+        }
+    }
+
+    get getClientAccountsRequestPath () {
+        return `${this.apiPrefix}/v1/client-info`
+    }
+
+    /**
+     * @typedef StatementTransactions
+     * @property {Array[Account]} accounts - optional
+     * @example
+     *
+     */
+
+    /**
+     * Get state of payment request, posted with `postPaymentRequest`
+     *
+     * @return {Promise<{error: any}|{data: StatementTransactions}>}
+     */
+    async getStatementTransactions (accountNumber, statementDate, page = 1) {
+        const { data, statusCode } = await this.request({
+            method: 'GET',
+            path: this.getStatementTransactionsRequestPath,
+            body: {
+                accountNumber,
+                statementDate,
+                page,
+            },
+        })
+        const jsonData = JSON.parse(data)
```

Is that method always return JSON as result? Maybe add check for it?

---

try/catch

---

Agree with @akarjakin.
It's better to throw an error in case when response cannot be parsed to JSON.